src/common.rs

```diff
@@ -4,22 +4,28 @@ use std::path::PathBuf;
 
 #[derive(Clone, Debug, PartialEq)]
 pub struct Warning {
+    pub check_name: String,
```

Comment:
Why is pub field? 
Perhaps this is necessary for another PR. I just don’t see this use right now

Reply:
Yes, in fact, this PR is splitted from another one.
I will remove `pub`.

Reply:
Done